.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,23 @@
+name: 💡 Deno land Feature request
+description: Suggest an idea, feature or enhancement
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting an idea. It helps make Deno better.
+
+        This is only for CLI / Deno land items. For community projects, please use the Ideas forum in [Discord](https://discord.com/deno).
```

Comment:
what is meant by "Deno land" here?

Reply:
Is it's aimed at official deno products 

I got that wording from the Ideas discord forum which makes such distinction

Reply:
well, this repo is only the deno CLI (and some of its crates) and not any of our other projects and products

Reply:
I've adjusted the text to remove the "deno land" term